twittermap/app/views/twittermap/main.scala.html

```diff
@@ -0,0 +1,48 @@
+@(title: String, ws: String)(body: Html)
+
+<!DOCTYPE html>
+<html ng-app="cloudberry">
+
+<head>
+  <title>@title</title>
+  <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link rel='shortcut icon' type='image/png' href='@routes.Assets.versioned("images/hyrax.png")'>
+
+  <link rel="stylesheet" media="screen" href='@routes.Assets.versioned("lib/bootstrap/css/bootstrap.css")'>
+  <link rel="stylesheet" media="screen" href='@routes.Assets.versioned("lib/leaflet/leaflet.css")'>
+  <link rel="stylesheet" media="screen" href='@routes.Assets.versioned("lib/dc.js/dc.min.css")'>
+  <link rel="stylesheet" media="screen" href='@routes.Assets.versioned("lib/font-awesome/css/font-awesome.min.css")'>
+  <link rel="stylesheet" media="screen" href='@routes.Assets.versioned("lib/bootstrap-vertical-tabs/bootstrap.vertical-tabs.min.css")'>
+  <link rel="stylesheet" media="screen" href='@routes.Assets.versioned("lib/bootstrap-toggle/css/bootstrap-toggle.min.css")'>
+  <link rel='stylesheet' href='@routes.Assets.versioned("stylesheets/main.css")'>
+
+  <script>
+    var config= {
```

Comment:
I want to keep one config file (`application.conf`) using the `Play` way. then I need to transfer the variable to the html page, and also let angular know it.